src/Components/test/E2ETest/Tests/ErrorNotificationClientSideTest.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using BasicTestApp;
+using Microsoft.AspNetCore.Components.E2ETest.Infrastructure;
+using Microsoft.AspNetCore.Components.E2ETest.Infrastructure.ServerFixtures;
+using Microsoft.AspNetCore.E2ETesting;
+using OpenQA.Selenium;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.AspNetCore.Components.E2ETest.Tests
+{
+    public class ErrorNotificationClientSideTest : ServerTestBase<ToggleExecutionModeServerFixture<Program>>
```

Comment:
I don't think it should be necessary to have separate `ErrorNotificationClientSideTest` and `ErrorNotificationServerSideTest`. The logic in them seems to be basically the same, and in theory I'd expect the same test to take the same actions and observe the same effects on both client and server.

Most of the E2E test classes are shared between client and server by means of subclassing. Could you check the pattern we use for other E2E test classes and do the same?

Or if I'm missing something and they have to be separate for some reason, could you clarify why?